DotNet.DocsTools/GitHubCommunications/MarkdownToHTMLRequest.cs

```diff
@@ -1,19 +1,13 @@
-﻿namespace DotnetDocsTools.GitHubCommunications;
+﻿using System.Text.Json;
+
+namespace DotnetDocsTools.GitHubCommunications;
```

Comment:
I'd probably revert this entire file too, as it is more complete than the proposed changes.

Reply:
It may come back, but reverted and commented out the one API where it is called.